### PR TITLE
Ports new evangelion jumpsuit sprites from bee, adds them to robotics vendor

### PR DESCRIPTION
**Pull request by SmArtKar**
## About The Pull Request

Resprites evangelion jumpsuits, puts them into robotics vendor's contraband section and makes them use skin system instead of being subtypes.
Oh, and adds a new jumpsuit to that selection.

Everything is done with Entropy's approval/request

image
image

Credits to https://github.com/BeeStation/BeeStation-Hornet/pull/5820 for sprites
image

## Changelog

:cl:
expansion: Evangelion jumpsuits have been added to robotics vendor's contraband section
imageadd: Ports new evangelion jumpsuit sprites
/:cl:

### CODE REVIEW

File: code/modules/clothing/under/costume.dm [hunk 1]
```diff
@@ -229,17 +229,11 @@
...
-	inhand_icon_state = "white_mech_suit"
-
-/obj/item/clothing/under/costume/mech_suit/blue
-	name = "blue mech pilot's suit"
-	desc = "A blue mech pilot's suit. For the more reluctant mech pilots."
-	icon_state = "blue_mech_suit"
-	inhand_icon_state = "blue_mech_suit"
+	unique_reskin = list("Red" = "red_mech_suit",
+						"White" = "white_mech_suit",
+						"Blue" = "detective_police",
```
Comment (Fikou) [inline code/modules/clothing/under/costume.dm:234]: detective police??

**Comment by Fikou** [comment]
you could make the create have 4 so 4 people could each have an unique color

File: code/modules/clothing/under/costume.dm (hunk 1 above)
Comment (SmArtKar) [reply code/modules/clothing/under/costume.dm:234]: the fuck, I thought I fixed that

File: code/modules/clothing/under/costume.dm [hunk 2]
```diff
@@ -229,17 +229,11 @@
...
-
-/obj/item/clothing/under/costume/mech_suit/blue
-	name = "blue mech pilot's suit"
-	desc = "A blue mech pilot's suit. For the more reluctant mech pilots."
-	icon_state = "blue_mech_suit"
-	inhand_icon_state = "blue_mech_suit"
+	unique_reskin = list("Red" = "red_mech_suit",
+						"White" = "white_mech_suit",
+						"Blue" = "blue_mech_suit",
+						"Retro" = "robotics_retro"
```
Comment (Fikou) [inline code/modules/clothing/under/costume.dm:235]: this isnt really retro, maybe for bee it is, id name it black or something

File: code/modules/clothing/under/costume.dm [hunk 3]
```diff
@@ -229,17 +229,11 @@
...
-	desc = "A white mech pilot's suit. Very fetching."
-	icon_state = "white_mech_suit"
-	inhand_icon_state = "white_mech_suit"
-
-/obj/item/clothing/under/costume/mech_suit/blue
-	name = "blue mech pilot's suit"
-	desc = "A blue mech pilot's suit. For the more reluctant mech pilots."
-	icon_state = "blue_mech_suit"
-	inhand_icon_state = "blue_mech_suit"
+	unique_reskin = list("Red" = "red_mech_suit",
```
Comment (Fikou) [inline code/modules/clothing/under/costume.dm:232]: dont format lists like that
do
= list(
  a,
  b,
)

File: code/modules/cargo/packs.dm [hunk 4]
```diff
@@ -2343,11 +2343,12 @@
 
 /datum/supply_pack/costumes_toys/mech_suits
 	name = "Mech Pilot's Suit Crate"
-	desc = "Suits for piloting big robots. Contains all three colors!"
+	desc = "Suits for piloting big robots. Contains three of those!"
```
Comment (Fikou) [inline code/modules/cargo/packs.dm:2346]: four

**Comment by SmArtKar** [comment]
image
here are your pictures citrus man

**Comment by Serxule** [comment]
oh are these those mech suits that didnt actually have sprites? or had missing ones

**Comment by SmArtKar** [comment]
> oh are these those mech suits that didnt actually have sprites? or had missing ones

No they just had shitty ones

**Comment by bluetshirtguy** [comment]
pictured: yet more anime references that will be merged asap.

**Comment by SmArtKar** [comment]
> pictured: yet more anime references that will be merged asap.

For those who don't know, this is just a sprite update. Previous versions of these also were ripped off from Evangelion

**Comment by Kylerace** [comment]
who created the sprites youre porting?

**Comment by SmArtKar** [comment]
> who created the sprites youre porting?

https://github.com/Wilchenx , link to his PR is literally in the PR description

**Comment by Fikou** [comment]
> who created the sprites youre porting?

they were commissioned by wilchen

**Review by Twaticus — APPROVED** [review]

**Pull request merged**